crates/test_probe/src/main.rs

```diff
@@ -0,0 +1,33 @@
+use composer::api::{Client, Event};
+use eyre::Result;
+use std::{env, thread, time::Duration};
+
+fn main() -> Result<()> {
+    color_eyre::install()?;
+
+    // Get server address, if given
+    let args: Vec<String> = env::args().collect();
```

Comment:
Nit: not super urgent, but I like using `clap` for all the argument handling so we get all the bells and whistles like usage strings, --help menu, etc :)

Reply:
Let's do this in a sperate PR. I filled an issue for myself https://github.com/tonarino/acoustic_profiler/issues/16